pkg/proxy/ipvs/proxier.go

```diff
@@ -1145,10 +1145,12 @@ func (proxier *Proxier) syncProxyRules() {
 	}
 	if !proxier.lbWhiteListCIDRSet.isEmpty() || !proxier.lbWhiteListIPSet.isEmpty() {
 		// link kube-services chain -> kube-fire-wall chain
-		args := []string{"-m", "set", "--match-set", proxier.lbIngressSet.Name, "dst,dst", "-j", string(KubeFireWallChain)}
-		if _, err := proxier.iptables.EnsureRule(utiliptables.Append, utiliptables.TableNAT, kubeServicesChain, args...); err != nil {
-			glog.Errorf("Failed to ensure that ipset %s chain %s jumps to %s: %v", proxier.lbIngressSet.Name, kubeServicesChain, KubeFireWallChain, err)
+		args := []string{
+			"-A", string(kubeServicesChain),
+			"-m", "set", "--match-set", proxier.lbIngressSet.Name,
+			"dst,dst",
 		}
+		writeLine(proxier.natRules, append(args, "-j", string(KubeFireWallChain))...)
```

Comment:
remove: `"-j", string(KubeFireWallChain))...` make it more consistent with other places?

Reply:
Sure.

Reply:
Sure